pkg/build/linux.go

```diff
@@ -216,6 +223,20 @@ func kernelBin(arch string) string {
 	}
 }
 
+var linuxCompilerRegexp = regexp.MustCompile("#define\\s+LINUX_COMPILER\\s+\"(.*)\"")
+
+func queryLinuxCompiler(kernelDir string) (string, error) {
+	bytes, err := ioutil.ReadFile(filepath.Join(kernelDir, "/include/generated/compile.h"))
```

Comment:
The proper way to express this is:
`filepath.Join(kernelDir, "include", "generated", "compile.h")`
Theoretically this can run on Windows, where path separator is not /

Reply:
Done